WcaOnRails/spec/mailers/job_failure_mailer_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+RSpec.describe JobFailureMailer, type: :mailer do
+  describe "notify_admin_of_job_failure" do
+    let(:job) { Delayed::Job.new(id: 42) }
+    let(:exception) do
+      begin
+        raise "error!"
+      rescue Exception => e
+        return e
+      end
+    end
+    let(:mail) { JobFailureMailer.notify_admin_of_job_failure(job, exception) }
```

Comment:
Ah, thanks for commenting on this, I forgot about this. I actually did construct a new exception at first, but it didn't have a backtrace in it. See:

```
~/gitting/worldcubeassociation.org @kaladin> irb
irb(main):001:0> RuntimeError.new("error!")
=> #<RuntimeError: error!>
irb(main):002:0> RuntimeError.new("error!").backtrace
=> nil
irb(main):003:0> error = nil; begin; raise "error!"; rescue Exception => e; error = e; end; error.backtrace
=> ["(irb):3:in `irb_binding'", "/usr/lib/ruby/2.3.0/irb/workspace.rb:87:in `eval'", "/usr/lib/ruby/2.3.0/irb/workspace.rb:87:in `evaluate'", "/usr/lib/ruby/2.3.0/irb/context.rb:380:in `evaluate'", "/usr/lib/ruby/2.3.0/irb.rb:489:in `block (2 levels) in eval_input'", "/usr/lib/ruby/2.3.0/irb.rb:623:in `signal_status'", "/usr/lib/ruby/2.3.0/irb.rb:486:in `block in eval_input'", "/usr/lib/ruby/2.3.0/irb/ruby-lex.rb:246:in `block (2 levels) in each_top_level_statement'", "/usr/lib/ruby/2.3.0/irb/ruby-lex.rb:232:in `loop'", "/usr/lib/ruby/2.3.0/irb/ruby-lex.rb:232:in `block in each_top_level_statement'", "/usr/lib/ruby/2.3.0/irb/ruby-lex.rb:231:in `catch'", "/usr/lib/ruby/2.3.0/irb/ruby-lex.rb:231:in `each_top_level_statement'", "/usr/lib/ruby/2.3.0/irb.rb:485:in `eval_input'", "/usr/lib/ruby/2.3.0/irb.rb:395:in `block in start'", "/usr/lib/ruby/2.3.0/irb.rb:394:in `catch'", "/usr/lib/ruby/2.3.0/irb.rb:394:in `start'", "/usr/bin/irb:11:in `<main>'"]

```